api/solarperformanceinsight_api/models.py

```diff
@@ -352,6 +352,47 @@ class SAPMTemperatureParameters(SPIBase):
     _modelchain_temperature_model: str = PrivateAttr("sapm")
 
 
+class NOCTSAMTemperatureParameters(SPIBase):
+    """Parameters for the NOCT SAM temperature model"""
+
+    noct: float = Field(
+        ...,
+        description=(
+            "Nominal operating cell temperature [C], determined at conditions of "
+            "800 W/m^2 irradiance, 20 C ambient air temperature and 1 m/s wind."
+        ),
+    )
+    eta_m_ref: float = Field(
+        ...,
+        description=(
+            "Module external efficiency [unitless] at reference conditions of "
+            "1000 W/m^2 and 20C."
+        ),
+    )
+    transmittance_absorptance: float = Field(
+        0.9,
+        description=(
+            "Coefficient for combined transmittance and absorptance effects. [unitless]"
+        ),
+    )
+    array_height: int = Field(
+        1,
+        description=(
+            "Height of array above ground in stories (one story is about 3m). Must "
+            "be either 1 or 2. For systems elevated less than one story, use 1. "
+            "If system is elevated more than two stories, use 2."
+        ),
+    )
```

Comment:
I'm just realizing that we probably need to be more specific here. pvlib really does require 1 or 2

```python
# pvlib.temperature.noct_sam
    if array_height == 1:
        wind_adj = 0.51 * wind_speed
    elif array_height == 2:
        wind_adj = 0.61 * wind_speed
    else:
        raise ValueError(
            f'array_height must be 1 or 2, {array_height} was given')
```